### PR TITLE
boot-utils: Add RISC-V to use '--use-cbl-qemu'

### PR DESCRIPTION
**Pull request by nathanchance**
Part 2 of resolving https://github.com/ClangBuiltLinux/continuous-integration2/issues/72

### CODE REVIEW

File: boot-qemu.sh [hunk 1]
```diff
@@ -238,6 +226,18 @@ function setup_qemu_args() {
             esac
             ;;
     esac
+    if ${USE_CBL_QEMU:-false} && [[ ${ARCH} = "riscv" || ${ARCH} = "s390" ]]; then
+        QEMU_BINARIES=${BASE}/qemu-binaries
+
+        green "Downloading or updating qemu-binaries..."
+        [[ -d ${QEMU_BINARIES} ]] || git clone https://github.com/ClangBuiltLinux/qemu-binaries "${QEMU_BINARIES}"
+        git -C "${QEMU_BINARIES}" pull --rebase
```
Comment (nickdesaulniers) [inline boot-qemu.sh:234]: consider shallow clone and `git reset --hard FETCH_HEAD`? Though probably overkill as we only have like 2 commits in that repo...it is of binaries though.

**Review by nickdesaulniers — APPROVED** [review]
LGTM thanks 🖲️ 🦠

**Comment by nathanchance** [comment]
Yes, I can double back to shallow cloning in the future. Thanks for the review!

**Comment by nathanchance** [comment]
Yes, I can double back to shallow cloning in the future. Thanks for the review!

**Pull request merged**